docs/articles/new-expensify/get-paid-back/Distance-Requests.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Distance Requests
+description: How to create a distance request and request reimbursement for mileage
+---
+<!-- The lines above are required by Jekyll to process the .md file -->
+
+# Overview
+
+Expensify allows you to request reimbursement for mileage by creating a distance request from a map. You can send a distance request in Expensify's mobile, desktop, or web app.
+
+
+# How to create and send a distance request
+
+1. Click the green + button and select Request Money.
+2. Select Distance along the top row of the Request Money window.
+3. Enter the Start and Finish addresses then click Next. If there are multiple stops, you can add them before clicking Next.
+4. Choose who to send the request to by selecting your organization's workspace from the list of recent workspaces.
+5. On the confirmation page, confirm the amount, date, distance, and optionally add a description or category. Click the Request button.
+6. A workspace admin will receive your request and can reimburse you through Expensify or elsewhere!
+
+
+
+# FAQs
```

Comment:
NAB: I didn't expect you to delete the whole question as mentioned [here](https://github.com/Expensify/App/pull/31668#discussion_r1410123150) 🙃 , but it's ok either way.

Reply:
Yeah, I tend to agree that was a good Q. :)

Reply:
Yeah, I mean I think you were right - Users can't submit mileage requests to group policies yet, and mileage rates can only be modified on group policies, so that question isn't relevant yet until you can submit requests to paid policies.

Reply:
FWIW, you can modify the mileage rate and unit in the free plan workspace editor on NewDot.